app/services/hybrid_transcription_service.py

```
@@ -0,0 +1,361 @@
+"""
+Hybrid Transcription Service - Final Version
+Simple strategy: YouTube API first → RunPod Whisper as fallback
+"""
+
+import time
+from typing import Dict, Any, Optional, Tuple
+from urllib.parse import urlparse, parse_qs
+
+from app.utils.logging_utils import get_logger
+from app.services.transcript_service import YouTubeTranscriptService, TranscriptExtractionError
```

Ruff (0.11.9)

`app.services.transcript_service.TranscriptExtractionError` imported but unused

Remove unused import: `app.services.transcript_service.TranscriptExtractionError`

(F401)